src/content/docs/en/guides/upgrade-to/v6.mdx

```diff
@@ -97,6 +97,30 @@ The following deprecated features are no longer supported and are no longer docu
 
 Some deprecated features may temporarily continue to function until they are completely removed. Others may silently have no effect, or throw an error prompting you to update your code.
 
+### `Astro` in `getStaticPaths()`
+
+Until now, it was possible to access the `Astro` object in `getStaticPaths()`. However, despite being typed the same as the `Astro` object accessible in the frontmatter, this object only had `site` and `generator` properties.
+
+In Astro 6.0, accessing `Astro.site` or `Astro.generator` within `getStaticPaths()` will log a warning. Accessing any other property will throw an error. In a future major version, accessing `Astro.site` or `Astro.generator` will also throw an error.
+
+#### What should I do?
+
+Remove usage of `Astro.generator` and update all occurrences of `Astro.site` in `getStaticPaths()`:
+
+```astro title="src/pages/blog/[slug].astro" del={5,6} ins={7}
+---
+import { getPages } from "../../../utils/data";
+
+export async function getStaticPaths() {
```

Comment:
In practice, I think pretty much nobody does this btw. So I struggled to find a meaningful example

Reply:
Then maybe a "realistic" example is the best we can hope for! I think this is fine!